doc/pprof.md

```diff
@@ -105,10 +105,50 @@ number of values - 1) or the name of the sample value.
 
 Sample values are numeric values associated to a unit. If pprof can recognize
 these units, it will attempt to scale the values to a suitable unit for
-visualization. The `unite=` option will force the use of a specific unit. For
+visualization. The `unit=` option will force the use of a specific unit. For
 example, `sample_index=sec` will force any time values to be reported in
 seconds. pprof recognizes most common time and memory size units.
 
+### Tag Filtering
+Samples in a profile may have tags. These tags have a name and a value; this
+value can be either numeric or a string. pprof can select samples from a profile
+based on these tags using the `-tagfocus` and `-tagignore` options.
+
+Generally, these options work as follows:
+* **-tagfocus= _regex_** or **-tagfocus= _range_:** Restrict to samples with tags
+  matched by regexp or in range
+* **-tagignore= _regex_** or **-tagignore= _range_:** Discard samples with tags 
+  matched by regexp or in range
+
+When using `-tagfocus= regex` and `-tagignore= regex`, the regex will be compared
+to `tagName:tagValue` for each value associated with the tag. If one specifies a regex
```

Comment:
I would maybe omit mentioning that we match against tag:value string. I view that as something we keep for backward compatibility for now, so maybe don't advertise.